src/test/net_tests.cpp

```diff
@@ -771,18 +771,22 @@ BOOST_AUTO_TEST_CASE(LimitedAndReachable_Network)
     SetReachable(NET_IPV4, false);
     SetReachable(NET_IPV6, false);
     SetReachable(NET_ONION, false);
+    SetReachable(NET_I2P, false);
 
     BOOST_CHECK_EQUAL(IsReachable(NET_IPV4), false);
     BOOST_CHECK_EQUAL(IsReachable(NET_IPV6), false);
     BOOST_CHECK_EQUAL(IsReachable(NET_ONION), false);
+    BOOST_CHECK_EQUAL(IsReachable(NET_I2P), false);
 
     SetReachable(NET_IPV4, true);
     SetReachable(NET_IPV6, true);
     SetReachable(NET_ONION, true);
+    SetReachable(NET_I2P, true);
 
     BOOST_CHECK_EQUAL(IsReachable(NET_IPV4), true);
     BOOST_CHECK_EQUAL(IsReachable(NET_IPV6), true);
     BOOST_CHECK_EQUAL(IsReachable(NET_ONION), true);
+    BOOST_CHECK_EQUAL(IsReachable(NET_I2P), true);
```

Comment:
nit:
```cpp
BOOST_CHECK_EQUAL(foo, true);
BOOST_CHECK_EQUAL(bar, false);
// vs
BOOST_CHECK(foo);
BOOST_CHECK(!bar);
```

but the surrounding code is already using the former, so it is ok to leave it as it is.

Reply:
Updated the surrounding ones (and saw a missing NET_I2P assertion, added). Thanks!